test/phantom-driver

```diff
@@ -50,6 +50,7 @@ var sys = require('system');
 var onCheckpoint;
 var lastError;
 var waitTimeout;
+var didTimeout;
```

Comment:
This is clearly the right thing to do, but I am not sure if it actually changes anything.  Without this declaration, 'didTimeout' should be a implicit global variable, not a local variable in each of the functions that use it.